packages/buendia-dashboard/data/usr/share/buendia/dashboard/downtime

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2015 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+up_count=$(cat /var/log/buendia/buendia-warmup.log | grep '(end, status 0)' | wc -l)
```

Comment:
I'm not sure that `buendia-warmup.log` is a reliable source of data. `buendia-warmup` seems to run multiple times in parallel some times and I thought it truncated the logs from time to time. Having said, I prefer this to performing yet another health check :)